VisualStudio.gitignore

```diff
@@ -27,7 +27,8 @@ x86/
 [Aa][Rr][Mm]/
 [Aa][Rr][Mm]64/
 bld/
-[Bb]in/
+**/[Bb]in/*
+!**/[Bb]in/*.refresh
```

Comment:
Rather than always include the refresh file in gitignore, can we make this a commented out section for now with a link to the docs to explain why this is needed?

Reply:
My last commit makes this change optional.

Reply:
Saimairjet@gmail.com